Sources/Client/Model/Channel/Channel.swift

```diff
@@ -120,8 +120,12 @@ public final class Channel: Codable {
     public let didLoad: Bool
     /// Checks if the channel is watching by the client.
     public var isWatched: Bool { Client.shared.isWatching(channel: self) }
+    /// Naming strategy to generate a name and image for the channel based on members.
+    /// Only takes effect on direct messages where `extraData` is `nil`.
+    public var namingStrategy: ChannelNamingStrategy? = DefaultNamingStrategy(maxUserNames: 1)
```

Comment:
I'd like to suggest to rename it to something like: `directMessageNamingStrategy` to explicitly separate from the channel naming/extradata. Developers can miss the description of the variable.